src/preamble.js

```diff
@@ -2033,6 +2033,14 @@ Module['FS_createPreloadedFile'] = FS.createPreloadedFile;
 var cyberDWARFFile = '{{{ BUNDLED_CD_DEBUG_FILE }}}';
 #endif
 
+Module['quit'] = function(status, toThrow) {
```

Comment:
Let's move this to `shell.js`, and split it into the different environment types (i.e. Module.quit should be defined in the section for each environment separately, see how it works there).